docs/content/tutorials/kic/deploy-with-kic.md

```diff
@@ -313,13 +322,18 @@ To learn how to expose a UDP application using NGINX Plus Ingress Controller, se
 
 Deploy NGINX Service Mesh with `mtls-mode` set to `off` and follow the [instructions](https://docs.nginx.com/nginx-ingress-controller/installation) to deploy NGINX Plus Ingress Controller.
 
-Add the enable-ingress and/or the enable-egress annotation shown below to the NGINX Plus Ingress Controller Pod spec:
+Add the enable-ingress and/or the enable-egress label shown below to the NGINX Plus Ingress Controller Pod spec:
 
 ```bash
 nsm.nginx.com/enable-ingress: "true"
 nsm.nginx.com/enable-egress: "true"
 ```
 
+{{<note>}}
+In NGINX Service Mesh versions prior to v1.7.0, `nsm.nginx.com/enable-ingress: "true"` and `nsm.nginx.com/enable-egress: "true"` annotations were used instead of labels.
+Support for these annotations will be removed in a future release.
+{{</note>}}
+
```

Comment:
Screenshot of changes from local testing.
![Screen Shot 2022-12-30 at 11 50 57 AM](https://user-images.githubusercontent.com/50845045/210107632-cbec14ec-96dd-420d-82f7-32175bc914ed.png)